modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,240 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+.. Enter Fluent Bit configuration information for your third-party logging solution, as shown in the following sections:
 
-* _Containers_ filter on names of containers.
-* _Pods_ filter on the names of pods. This is useful for specifying application names followed by a wildcard (`%`).
+*** <<Azure Log Analytics Configuration Parameters, Azure Log Analytics Configuration Parameters>>
+*** <<Elasticsearch Configuration Parameters, Elasticsearch Configuration Parameters>>
+*** <<GELF Configuration Parameters, GELF Configuration Parameters>>
+*** <<Splunk Configuration Parameters, Splunk Configuration Parameters>>
 
-== Forward Logs to External Services
+.. Select *Submit*.
+.. To verify successful log forwarding, manually check the logs using the external log forwarding service.
+
+==== Azure Log Analytics Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Azure Log Analytics Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Customer_ID | Specifies the customer ID or workspace ID string. | Yes |
+| Shared_Key | Specifies the primary or secondary connected sources client authentication key. | Yes |
+| Log_Type | Specifies the event type name. | No | `fluentbit`
+|===
 
-Anypoint Runtime Fabric enables you to forward application and cluster logs to an external logging service. The log forwarder built in to Runtime Fabric enables you to send log data to an `rsyslog` server over TCP or UDP.
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name        azure
+    Match       *
+    Customer_ID id
+    Shared_Key  key
+```
+
+==== Elasticsearch Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Elasticsearch Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Specifies the IP address or hostname of the target Elasticsearch instance. | | 127.0.0.1
+| Port | Specifies the TCP port of the target Elasticsearch instance. | | 9200
+| Path | Elasticsearch accepts new data on HTTP query path `/_bulk`. But it is also possible to serve Elasticsearch behind a reverse proxy on a subpath. This option defines such a path on the Fluent Bit side. It adds a path prefix in the indexing HTTP POST URI. | | Empty string
+| Buffer_Size | Specifies the buffer size used to read the response from the Elasticsearch HTTP service. This option is useful for debugging purposes where reading the full response is needed. Note that the size of the response grows depending on the number of records inserted. To specify an unlimited amount of memory, set this value to `False`. Otherwise, set the value according to the Unit Size specification. | | 4KB
+| Pipeline | Newer versions of Elasticsearch provide setup filters called pipelines. This option specifies which pipeline the database should use. However, to optimize performance, you should perform parsing and filtering on the Fluent Bit side and avoid pipelines. | | 
```

Comment:
We remove `Pipeline`: 

> However, to optimize performance, you should perform parsing and filtering on the Fluent Bit side and avoid pipelines.

So i have also dropped this from API.

Reply:
`Pipeline` removed.